Task-6/Task-6/Services/InvoiceHandler.cs

```diff
@@ -0,0 +1,119 @@
+using Newtonsoft.Json;
+using NLog;
+using Task_6.Models;
+
+namespace Task_6.Services;
+
+public class InvoiceHandler
```

Comment:
Invoice is not working(Save choosen phone and create output file). Try to fix it